### ISSUE TITLE
Upgrading to 3.0.0 causes System.MissingMethodException: Method not found: 'Boolean Microsoft.VisualStudio.TestPlatform.CoreUtilities.Helpers.CommandLineArgumentsHelper.TryGetIntArgFromDict

### ISSUE THREAD

**Issue by abatishchev** [coverlet-coverage/coverlet#1042]
My service is built using Azure DevOps:

```
Current agent version: '2.175.2'
```

If I upgrade to version `3.0.0` my tests fail with error:

```
Starting: Running tests from ResourceProviderService.Tests.Functional
==============================================================================
Task         : .NET Core
Description  : Build, test, package, or publish a dotnet application, or run a custom dotnet command
Version      : 2.179.2
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/build/dotnet-core-cli
==============================================================================
C:\windows\system32\chcp.com 65001
Active code page: 65001
Info: .NET Core SDK/runtime 2.2 and 3.0 are now End of Life(EOL) and have been removed from all hosted agents. If you're using these SDK/runtimes on hosted agents, kindly upgrade to newer versions which are not EOL, or else use UseDotNet task to install the required version.
C:\dotnet\dotnet.exe test C:\azagent\_work\1\a\Service.Tests.Functional\Service.Tests.Functional.dll --logger trx --results-directory C:\azagent\_work\_temp --settings C:\azagent\_work\1\a/Service.Tests.Functional/RunSettings/FunctionalTests.Dev.RunSettings
Microsoft (R) Test Execution Command Line Tool Version 16.7.0
Copyright (c) Microsoft Corporation.  All rights reserved.

Starting test execution, please wait...

A total of 1 test files matched the specified pattern.
Testhost process exited with error: Unhandled exception. System.MissingMethodException: Method not found: 'Boolean Microsoft.VisualStudio.TestPlatform.CoreUtilities.Helpers.CommandLineArgumentsHelper.TryGetIntArgFromDict(System.Collections.Generic.IDictionary`2<System.String,System.String>, System.String, Int32 ByRef)'.
   at Microsoft.VisualStudio.TestPlatform.TestHost.DefaultEngineInvoker.SetParentProcessExitCallback(IDictionary`2 argsDictionary)
   at Microsoft.VisualStudio.TestPlatform.TestHost.DefaultEngineInvoker.Invoke(IDictionary`2 argsDictionary)
   at Microsoft.VisualStudio.TestPlatform.TestHost.Program.Run(String[] args)
   at Microsoft.VisualStudio.TestPlatform.TestHost.Program.Main(String[] args)
. Please check the diagnostic logs for more information.
Results File: C:\azagent\_work\_temp\testagent000000$_testagent000000_2021-01-12_19_59_37.trx
```

Rolling back to `1.3.0` makes the issue to disappear.

Other installed packages:

```xml
<ItemGroup>
  <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.3" />
  <PackageReference Include="MSTest.TestAdapter" Version="2.1.2" />
  <PackageReference Include="MSTest.TestFramework" Version="2.1.2" />
  <PackageReference Include="coverlet.collector" Version="1.3.0">
    <PrivateAssets>all</PrivateAssets>
    <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
  </PackageReference>
</ItemGroup>
```

**Comment by Kampfmoehre** [comment]
Discovering the same problem with Net 5. I have an integration test project that looks similar to the one from the [MS docs](https://docs.microsoft.com/en-us/aspnet/core/test/integration-tests?view=aspnetcore-5.0). Is there anything I can provide for further investigation of the issue?

```xml
  <ItemGroup>
    <PackageReference Include="coverlet.collector" Version="3.0.0">
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
      <PrivateAssets>all</PrivateAssets>
    </PackageReference>
    <PackageReference Include="coverlet.msbuild" Version="3.0.0">
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
      <PrivateAssets>all</PrivateAssets>
    </PackageReference>
    <PackageReference Include="JUnitTestLogger" Version="1.1.0" />
    <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="5.0.2" />
    <PackageReference Include="Microsoft.EntityFrameworkCore.InMemory" Version="5.0.2" />
    <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="5.0.2" />
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.3" />
    <PackageReference Include="Moq" Version="4.15.2" />
    <PackageReference Include="xunit" Version="2.4.1" />
    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3">
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
      <PrivateAssets>all</PrivateAssets>
    </PackageReference>
  </ItemGroup>
```

**Comment by MarcoRossignoli** [comment]
Can someone create a small repro?

**Comment by fredrikcarlbom** [comment]
> 
> 
> Can someone create a small repro?

https://github.com/fredrikcarlbom/coverlet

**Comment by MarcoRossignoli** [comment]
@fredrikcarlbom for now you can try with `dotnet test --collect:"XPlat Code Coverage"` your sample is working on my test. 

```
❯ dotnet test --collect:"XPlat Code Coverage"
  Determining projects to restore...
  All projects are up-to-date for restore.
  You are using a preview version of .NET. See: https://aka.ms/dotnet-core-preview
  You are using a preview version of .NET. See: https://aka.ms/dotnet-core-preview
  lib1 -> C:\git\coverletissue\coverlet\lib1\bin\Debug\net5.0\lib1.dll
  lib1.tests -> C:\git\coverletissue\coverlet\lib1.tests\bin\Debug\net5.0\lib1.tests.dll
Test run for C:\git\coverletissue\coverlet\lib1.tests\bin\Debug\net5.0\lib1.tests.dll (.NETCoreApp,Version=v5.0)
Microsoft (R) Test Execution Command Line Tool Version 16.9.0
Copyright (c) Microsoft Corporation.  All rights reserved.

Starting test execution, please wait...
A total of 1 test files matched the specified pattern.

Attachments:
  C:\git\coverletissue\coverlet\lib1.tests\TestResults\948060e5-f4c8-4a66-b4b2-df5b4e1e5af9\coverage.cobertura.xml
Passed!  - Failed:     0, Passed:     1, Skipped:     0, Total:     1, Duration: 2 ms - lib1.tests.dll (net5.0)
 C:\git\coverletissue\coverlet   main ≣ +0 ~1 -0 ! 
```
Investigating publish scenario

**Comment by MarcoRossignoli** [comment]
@fredrikcarlbom @abatishchev can you try with this preview version?

[Preview.zip](https://github.com/coverlet-coverage/coverlet/files/5860499/Preview.zip)

**Comment by Kampfmoehre** [comment]
This resolves it for me on my Fedora machine.

I installed it with
`dotnet add test/<ProjectName>.IntegrationTests/ package --source /home/<user>/Downloads/ --prerelease coverlet.collector`
and
`dotnet add test/<ProjectName>.IntegrationTests/ package --source /home/<user>/Downloads/ --prerelease coverlet.msbuild`

**Comment by MarcoRossignoli** [comment]
We released `3.0.2` last week should be same of attached preview package.

**Comment by artazar** [comment]
@MarcoRossignoli tests in our pipelines started failing with precisely the same stack trace today, but the version was bumped to 3.0.2 exactly (from 1.0.1). So could you confirm once again which version will contain the fix?

**Comment by MarcoRossignoli** [comment]
Mmm `3.0.2` is the last one, so I think that the issue is not solved. 
We need to investigate.

**Comment by blushingpenguin** [comment]
I get this on Linux (ubuntu 20.04) but not Windows 10 x64

**Comment by agsydney** [comment]
Same issue for me, I was getting the following in Azure Devops with coverlet 3.0.2:

2021-02-10T22:37:32.8278999Z ##[error]Testhost process exited with error: Unhandled exception. System.MissingMethodException: Method not found: 'Boolean Microsoft.VisualStudio.TestPlatform.CoreUtilities.Helpers.CommandLineArgumentsHelper.TryGetIntArgFromDict(System.Collections.Generic.IDictionary`2<System.String,System.String>, System.String, Int32 ByRef)'.
2021-02-10T22:37:32.8291747Z ##[error]   at Microsoft.VisualStudio.TestPlatform.TestHost.DefaultEngineInvoker.SetParentProcessExitCallback(IDictionary`2 argsDictionary)
2021-02-10T22:37:32.8293143Z ##[error]   at Microsoft.VisualStudio.TestPlatform.TestHost.DefaultEngineInvoker.Invoke(IDictionary`2 argsDictionary)
2021-02-10T22:37:32.8294283Z ##[error]   at Microsoft.VisualStudio.TestPlatform.TestHost.Program.Run(String[] args)
2021-02-10T22:37:32.8295380Z ##[error]   at Microsoft.VisualStudio.TestPlatform.TestHost.Program.Main(String[] args)

Once I reverted back to 1.3.0 it worked fine again.

Let me know if you'd like any more details

**Comment by daveMueller** [comment]
Could someone do me a favor and test the attached collector nupkg? 

[Preview.Packages.zip](https://github.com/coverlet-coverage/coverlet/files/6032836/Preview.Packages.zip)

With the repro @fredrikcarlbom provided this worked for me. 
@MarcoRossignoli I could reproduce the issue with @fredrikcarlbom repro and 3.0.3 collector. With the attached preview
it's working for me. But I only tested on Windows so far.

**Comment by MarcoRossignoli** [comment]
@daveMueller some spoiler on issue? New object model deps to vs plat?

**Comment by daveMueller** [comment]
> @daveMueller some spoiler on issue? New object model deps to vs plat?

Yes exactly.

**Comment by MarcoRossignoli** [comment]
@daveMueller we need to do some test also for netcoreapp2.x to understand if it works also there

**Comment by daveMueller** [comment]
OK I  will change the target framework on the repro and also do some tests on linux.

**Comment by MarcoRossignoli** [comment]
I mean if the fix in that case is only upgrade `Microsoft.NET.Test.Sdk` to have a compat object model with new one(new version used by coverlet collectors) of project running tests for netcoreapp2.x it's ok.

**Comment by daveMueller** [comment]
So I now also did some tests on linux. I could reproduce the issue and using the preview collector package I provided fixed it. 
I also changed the target framework to ```netcoreapp2.1``` and didn't have any problems.

@MarcoRossignoli The only change I made was an upgrade of ```Microsoft.TestPlatform.ObjectModel``` to version 16.9.1.

**Comment by MarcoRossignoli** [comment]
Does it work also with netcoreapp2.0 and net4xx? What is the minimum version of Microsoft.NET.Test.Sdk that use it?So we can also update the docs.

**Comment by daveMueller** [comment]
OK I'm a bit confused now. I can't get it running on ```netcoreapp2.0```. But I also can't get the ```coverlet.collector``` v1.3.0 running on ```netcoreapp2.0``` target framework. In the documentation ([doc](https://github.com/coverlet-coverage/coverlet#requirements)) it says min .NET Core SDK v2.2.401?

As for the ```Microsoft.NET.Test.Sdk``` the min version in our documentation (16.5.0) is working.
I still have to test full framework.

**Comment by MarcoRossignoli** [comment]
> OK I'm a bit confused now. I can't get it running on netcoreapp2.0. But I also can't get the coverlet.collector v1.3.0 running on netcoreapp2.0 target framework. In the documentation (doc) it says min .NET Core SDK v2.2.401?

You're right, completely forgot it 😄 we can test only for >= 2.2

**Comment by daveMueller** [comment]
OK I now finished all my test and for me this seems to be OK. I also tested for ```net472``` but here the issue doesn't exit whether using ```coverlet.collector``` v1.3.0 or v3.0.0. Seems to be core related.

**Comment by MarcoRossignoli** [comment]
> v3.0.0

Collectors supports netxxx  should be testable https://github.com/coverlet-coverage/coverlet/blob/master/Documentation/DriversFeatures.md

**Comment by MarcoRossignoli** [comment]
@abatishchev @Kampfmoehre @fredrikcarlbom @artazar @blushingpenguin @agsydney can someone give it a try and tell us if it works as expected?
Will be on nightly tomorrow https://github.com/coverlet-coverage/coverlet/blob/master/Documentation/ConsumeNightlyBuild.md

**Comment by Kampfmoehre** [comment]
Unfortunately the problem appears again with the nightly version:

```
/home/<user>/.nuget/packages/coverlet.msbuild/3.0.0-preview.18.g183cbed8a6/build/coverlet.msbuild.targets(39,5): warning : [coverlet] Unable to resolve method reference "Microsoft.Extensions.Hosting.IHost Microsoft.Extensions.Hosting.IHostBuilder::Build()", assuming calls to will return [/home/<user>/dev/<project>/test/<ProjectName>.UnitTest/<ProjectName>.UnitTest.csproj]
/home/<user>/.nuget/packages/coverlet.msbuild/3.0.0-preview.18.g183cbed8a6/build/coverlet.msbuild.targets(39,5): warning : [coverlet] Unable to resolve method reference "System.Void Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions::Run(Microsoft.Extensions.Hosting.IHost)", assuming calls to will return [/home/<user>/dev/<project>/test/<ProjectName>.UnitTest/<ProjectName>.UnitTest.csproj]
/home/<user>/.nuget/packages/coverlet.msbuild/3.0.0-preview.18.g183cbed8a6/build/coverlet.msbuild.targets(39,5): warning : [coverlet] Unable to resolve method reference "Microsoft.Extensions.Hosting.IHostBuilder Microsoft.Extensions.Hosting.Host::CreateDefaultBuilder(System.String[])", assuming calls to will return [/home/<user>/dev/<project>/test/<ProjectName>.UnitTest/<ProjectName>.UnitTest.csproj]
```

The problem was fixed for me with the 3.0.3 packages. I tried both the msbuild and the collector package with the nightly version and both are giving me a lot of messages like the above.

I am running on Fedora with .NET 5.

**Comment by daveMueller** [comment]
@Kampfmoehre  Thanks for the test. Unfortunately I can't reproduce it with the nightly anymore which makes this even harder. I tested on Windows and Ubuntu but I will try to analyze this again and set up a Fedora OS.

**Comment by Kampfmoehre** [comment]
I am not sure if it has anything to do with Fedora, but to help to further narrow it down, I am using the dotnet-sdk package from microsoft prod repo instead of the ones from Fedoras own repo.
I had some problems with the the packages from the Fedora repo - or more with a mix of packages from both repos:
https://github.com/OmniSharp/omnisharp-vscode/issues/4360#issuecomment-779086868
https://github.com/dotnet/sdk/issues/15863#issuecomment-783262435

TL;DR install dotnet like in the [official Docs](https://docs.microsoft.com/en-gb/dotnet/core/install/linux-fedora#install-net-50).

I don't think that this has to do with the issue here but I can't rule it out either. The problems with the collector came with the 3.0.0 version and were resovled with the preview you gave us in late january. Since then I upgraded to 3.0.3 (we use renovate to keep our packages up-to-date) and didn't have any other problems. Just for the test I used the new nightly ones and the problems appeared again. They vanished once I got back to 3.0.3.

If there is any additional info I can provide, just tell me.

**Comment by daveMueller** [comment]
Yes I also don't really think it is the different distro but I can't really figure out what the difference is that causes this issue. Your other package references also lock similar to mine. If you find some time could you try to reproduce it with the repo @fredrikcarlbom provided [https://github.com/fredrikcarlbom/coverlet](https://github.com/fredrikcarlbom/coverlet)?

**Comment by Kampfmoehre** [comment]
I cloned the repo and run `build.sh`, this is the output:
```
./build.sh
Microsoft (R) Build Engine version 16.8.3+39993bd9d for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

  Determining projects to restore...
  Restored /home/<user>/dev/coverlet/lib1/lib1.csproj (in 115 ms).
  Restored /home/<user>/dev/coverlet/lib1.tests/lib1.tests.csproj (in 2.44 sec).
  lib1 -> /home/<user>/dev/coverlet/lib1/bin/Release/net5.0/lib1.dll
  lib1.tests -> /home/<user>/dev/coverlet/lib1.tests/bin/Release/net5.0/lib1.tests.dll
  lib1.tests -> /home/<user>/dev/coverlet/out/
~/dev/coverlet/out ~/dev/coverlet
Microsoft (R) Test Execution Command Line Tool Version 16.8.3
Copyright (c) Microsoft Corporation.  All rights reserved.

Starting test execution, please wait...
A total of 1 test files matched the specified pattern.
Testhost process exited with error: Unhandled exception. System.MissingMethodException: Method not found: 'Boolean Microsoft.VisualStudio.TestPlatform.CoreUtilities.Helpers.CommandLineArgumentsHelper.TryGetIntArgFromDict(System.Collections.Generic.IDictionary`2<System.String,System.String>, System.String, Int32 ByRef)'.
   at Microsoft.VisualStudio.TestPlatform.TestHost.DefaultEngineInvoker.SetParentProcessExitCallback(IDictionary`2 argsDictionary)
   at Microsoft.VisualStudio.TestPlatform.TestHost.DefaultEngineInvoker.Invoke(IDictionary`2 argsDictionary)
   at Microsoft.VisualStudio.TestPlatform.TestHost.Program.Run(String[] args)
   at Microsoft.VisualStudio.TestPlatform.TestHost.Program.Main(String[] args)
. Please check the diagnostic logs for more information.

Attachments:
  /home/<user>/dev/coverlet/out/TestResults/93303ec2-ad08-473e-902c-c80685932d3b/coverage.cobertura.xml
Test Run Aborted.
```

References:
<details>
  <summary>coverage.xml content</summary>
The referenced xml file content is this:

```xml
<?xml version="1.0" encoding="utf-8"?>
<coverage line-rate="1" branch-rate="1" version="1.9" timestamp="1614677035" lines-covered="0" lines-valid="0" branches-covered="0" branches-valid="0">
  <sources />
  <packages>
    <package name="lib1" line-rate="1" branch-rate="1" complexity="0">
      <classes />
    </package>
  </packages>
</coverage>
```
</details>

<details>
  <summary>csproj content</summary>
For reference this is the content of my current test project csproj file which is working:

```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>net5.0</TargetFramework>
    <NoWarn>SA0001</NoWarn>
    <IsPackable>false</IsPackable>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="coverlet.collector" Version="3.0.3" />
    <PackageReference Include="coverlet.msbuild" Version="3.0.3">
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
      <PrivateAssets>all</PrivateAssets>
    </PackageReference>
    <PackageReference Include="FluentAssertions" Version="5.10.3" />
    <PackageReference Include="JUnitTestLogger" Version="1.1.0" />
    <PackageReference Include="Microsoft.EntityFrameworkCore.InMemory" Version="5.0.3" />
    <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="5.0.3" />
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.9.1" />
    <PackageReference Include="Moq" Version="4.16.1" />
    <PackageReference Include="xunit" Version="2.4.1" />
    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3" />
  </ItemGroup>

  <ItemGroup>
    <ProjectReference Include="..\..\src\<projectName>\<projectName>.csproj" />
  </ItemGroup>

</Project>
```
</details>

<details>
  <summary>The `dotnet --info` output:</summary>

```
.NET SDK (reflecting any global.json):
 Version:   5.0.103
 Commit:    9effbc8ad5

Runtime Environment:
 OS Name:     fedora
 OS Version:  33
 OS Platform: Linux
 RID:         fedora.33-x64
 Base Path:   /usr/share/dotnet/sdk/5.0.103/

Host (useful for support):
  Version: 5.0.3
  Commit:  eae88cc11b

.NET SDKs installed:
  3.1.406 [/usr/share/dotnet/sdk]
  5.0.103 [/usr/share/dotnet/sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 3.1.12 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.3 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 3.1.12 [/usr/share/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.3 [/usr/share/dotnet/shared/Microsoft.NETCore.App]

To install additional .NET runtimes or SDKs:
  https://aka.ms/dotnet-download
```

</details>

**Comment by daveMueller** [comment]
@Kampfmoehre  Sorry to ask this but did you update the repo to use the nightly?

**Comment by Kampfmoehre** [comment]
Nope sorry I forgot. But I just did and the message remains the same:
<details>
  <summary>Build error</summary>

```
Microsoft (R) Build Engine version 16.8.3+39993bd9d for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

  Determining projects to restore...
  Restored /home/<user>>/dev/coverlet/lib1/lib1.csproj (in 136 ms).
  Restored /home/<user>>/dev/coverlet/lib1.tests/lib1.tests.csproj (in 347 ms).
  lib1 -> /home/<user>>/dev/coverlet/lib1/bin/Release/net5.0/lib1.dll
  lib1.tests -> /home/<user>>/dev/coverlet/lib1.tests/bin/Release/net5.0/lib1.tests.dll
  lib1.tests -> /home/<user>>/dev/coverlet/out/
~/dev/coverlet/out ~/dev/coverlet
Microsoft (R) Test Execution Command Line Tool Version 16.8.3
Copyright (c) Microsoft Corporation.  All rights reserved.

Starting test execution, please wait...
A total of 1 test files matched the specified pattern.
Testhost process exited with error: Unhandled exception. System.MissingMethodException: Method not found: 'Boolean Microsoft.VisualStudio.TestPlatform.CoreUtilities.Helpers.CommandLineArgumentsHelper.TryGetIntArgFromDict(System.Collections.Generic.IDictionary`2<System.String,System.String>, System.String, Int32 ByRef)'.
   at Microsoft.VisualStudio.TestPlatform.TestHost.DefaultEngineInvoker.SetParentProcessExitCallback(IDictionary`2 argsDictionary)
   at Microsoft.VisualStudio.TestPlatform.TestHost.DefaultEngineInvoker.Invoke(IDictionary`2 argsDictionary)
   at Microsoft.VisualStudio.TestPlatform.TestHost.Program.Run(String[] args)
   at Microsoft.VisualStudio.TestPlatform.TestHost.Program.Main(String[] args)
. Please check the diagnostic logs for more information.

Attachments:
  /home/<user>>/dev/coverlet/out/TestResults/e78030dd-2f34-4cd8-8399-7dfa817e8268/coverage.cobertura.xml
Test Run Aborted.
```
</details>

<details>
  <summary>csproj</summary>

```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>net5.0</TargetFramework>
    <IsPackable>false</IsPackable>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="coverlet.msbuild" Version="3.0.0-preview.18.g183cbed8a6">
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
      <PrivateAssets>all</PrivateAssets>
    </PackageReference>
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.0" />
    <PackageReference Include="xunit" Version="2.4.1" />
    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3">
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
      <PrivateAssets>all</PrivateAssets>
    </PackageReference>
    <PackageReference Include="coverlet.collector" Version="3.0.0">
      <PrivateAssets>all</PrivateAssets>
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
    </PackageReference>
  </ItemGroup>

  <ItemGroup>
    <ProjectReference Include="..\lib1\lib1.csproj" />
  </ItemGroup>

</Project>
```
</details>

**Comment by Kampfmoehre** [comment]
Ok my bad - apparently I had the wrong version installed since I copied the command from the docs you posted. I installed the latest nightly package with 
`dotnet add package coverlet.collector --prerelease --source https://pkgs.dev.azure.com/tonerdo/coverlet/_packaging/coverlet-nightly/nuget/v3/index.json` and now the warnings are gone. The version is now `3.0.4-preview.4.g5de0ad7d60`. Sorry for the inconvenience.
